phabricatoremails/config.py

```diff
@@ -0,0 +1,102 @@
+import configparser
+import smtplib
+from configparser import ConfigParser
+from dataclasses import dataclass
+from logging import Logger
+from typing import Any
+
+import psycopg2
+from phabricatoremails.logging import create_dev_logger, create_logger
+from phabricatoremails.mail import SesMail, SmtpMail, FsMail
+from phabricatoremails.progress import StaticProgress, PhabricatorProgress
+from phabricatoremails.source import FileSource, PhabricatorSource
+from psycopg2.extras import DictCursor
+
+
+def _parse_logger(config: ConfigParser):
+    if config.has_section("dev"):
+        return create_dev_logger()
+    else:
+        return create_logger()
```

Comment:
Hmm, I don't like the idea of throwing the config into global state, then depending on that in later calls. It feels vaguely "magical".
By tying it together, it makes it easier to understand. I want to avoid confusion of "how does `logging.getLogger(...)` know whether I want a dev logger or regular logger?"